rust_snuba/src/lib.rs

```diff
@@ -3,10 +3,48 @@ mod consumer;
 mod strategies;
 mod types;
 
-use pyo3::prelude::*;
+use pyo3::{prelude::*};
 
 #[pymodule]
 fn rust_snuba(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
+    setup_sentry(_py);
     m.add_function(wrap_pyfunction!(consumer::consumer, m)?)?;
+    m.add_function(wrap_pyfunction!(setup_sentry, m)?)?;
     Ok(())
 }
+
+#[pyfunction]
+pub fn setup_sentry(_py: Python<'_>) {
+    env_logger::init();
+    let sentry_dsn: Result<Option<String>, PyErr> = Python::with_gil(|py| {
+        let snuba_settings = PyModule::import(py, "snuba.settings")?;
```

Comment:
Instead of this I would only call `setup_sentry` in consumer::consumer, and use the existing config payload which we already use to pass consumer config, to pass the DSN.

Reply:
ok. I updated it to be that way.